lib/modules/datasource/types.ts

```diff
@@ -65,6 +65,7 @@ export interface ReleaseResult {
   changelogUrl?: string;
   dependencyUrl?: string;
   homepage?: string;
+  gitRef?: string;
```

Comment:
@rarkins WDYT?

Reply:
let me know if you need anything else from me to move forward with this!